lib/livebook/config.ex

```diff
@@ -493,4 +501,25 @@ defmodule Livebook.Config do
     IO.puts("\nERROR!!! [Livebook] " <> message)
     System.halt(1)
   end
+
+  @doc """
+  Parses zero trust identity provider from env.
+  """
+  def identity_provider!(env) do
+    zta = System.get_env(env) || "cookies"
+
+    case zta do
```

Comment:
We don't need a default since we only call this if the env var is present, right?

```suggestion
    case System.get_env(env) do
```

Reply:
It is in a conditional, no? `if identity_provider = Livebook.Config.identity_provider!("LIVEBOOK_IDENTITY_PROVIDER")`

Reply:
Ah nevermind, my bad.

Reply:
But for consistency with other env vars we should probably return `nil` for `nil` and do `|| {LivebookWeb.Cookies, :unused}` in the config file.